lms/envs/aws.py

```diff
@@ -1099,6 +1099,39 @@
 INSTALLED_APPS.append('openedx.features.ucsd_features')
 
 ############### Settings for Caliper Tracking #####################
+# 'caliper_tracking' app enables us to transform Edx event-logs according to
+# IMSGlobal Caliper Standards.
+# https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html.
+#
+# Following are the changes made on edx by caliper_tracking app.
```

Comment:
`Following are the changes made in Django settings by caliper_tracking app.`

Reply:
done